rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.69.0"
+channel = "1.70.0"
```

Comment:
This is the error I have with `1.69.0`, which is gone using `1.70.0`.

```
error: package `clap_builder v4.4.8` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.69.0
Either upgrade to rustc 1.70.0 or newer, or use
cargo update -p clap_builder@4.4.8 --precise ver
where `ver` is the latest version of `clap_builder` supporting rustc 1.69.0
```

Reply:
Can't say anything about the error, probably an issue with the lock file. One of the reasons for having the cargo.lock is for not using any libs which bring breaking change to the project as it is the case with `clap`. It's not a big issue to upgrade to 1.70.0, although it's a kind of breaking change. I will resolve this conversation